src/service-worker.ts

```diff
@@ -0,0 +1,74 @@
+/// <reference lib="webworker" />
```

Comment:
Is this supposed to be commented out?

Reply:
Is this file auto-generated?

Reply:
This is actually the syntax for Triple Slash Directives (see [documentation](https://www.typescriptlang.org/docs/handbook/triple-slash-directives.html)). This was autogenerated, so I don't have the strongest understanding about what this does, but my current understanding is that it's used by the TypeScript compiler.